pom.xml

```diff
@@ -88,7 +88,7 @@
     <url>https://github.com/snowdrop</url>
     <connection>scm:git:https://github.com/snowdrop/spring-boot-bom.git</connection>
     <developerConnection>scm:git:git@github.com:snowdrop/spring-boot-bom.git</developerConnection>
-    <tag>1.5.10.Beta1</tag>
+    <tag>1.5.13-SNAPSHOT</tag>
```

Comment:
That change doesn't matter as far as I'm aware.

Reply:
Wouldn't this make a release plugin automatically fix the version?

Reply:
I'm not sure how it works, actually but I think the release plugin modifies it when it tags so the value during the SNAPSHOT stage doesn't really matters.